federation/deploy/init/dep.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    run: fi
```

Comment:
Did you mean federation-init by fi? We encourage clearer labels rather than shorter ones.